test/helpers.py

```diff
@@ -281,26 +296,40 @@ def sde_solver_strong_order(
     return steps_arr, errs_arr, order
 
 
-@dataclasses.dataclass(frozen=True)
+@dataclasses.dataclass(unsafe_hash=True)
```

Comment:
Eek, it's unsafe for a reason. Why?

Reply:
Hmm I remember there being a problem with frozen, but not sure what. I'll change it back and then if all hell breaks loose we'll know why.